### PR TITLE
Be able to use Redis sentinel

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/vars.yaml_tmpl [hunk 1]
```diff
@@ -161,6 +161,18 @@ vars:
...
+  cache:
+    std:
+      arguments:
+        # Standalone version
+        # host: '{REDIS_HOST}'
+        # port: '{REDIS_PORT}'
+        # Openshift version
+        sentinels:
+          - - ${REDIS_HOST}
+              ${REDIS_PORT}
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/vars.yaml_tmpl:174]: Seems wreid, or I do not know this syntax `- -`

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/tilegeneration/config.yaml.tmpl_tmpl [hunk 2]
```diff
@@ -97,7 +97,15 @@ generation:
...
 
 redis:
-    url: 'redis://${REDIS_HOST}:${REDIS_PORT}/${REDIS_DB}'
+    socket_timeout: ${REDIS_TIMEOUT}
+    # For standalone version
+    # url: 'redis://${REDIS_HOST}:${REDIS_PORT}/${REDIS_DB}'
+    # For OpenShift version
+    sentinels:
+      - - ${REDIS_HOST}
+        - ${REDIS_PORT}
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/tilegeneration/config.yaml.tmpl_tmpl:106]: Same syntax `- -` but with `-` on second line.

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/tilegeneration/config.yaml.tmpl_tmpl:106]: In JSON it is: `[["host", "port"]]` :-)

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/vars.yaml_tmpl (hunk 1 above)
Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/vars.yaml_tmpl]: This one is wrong, one dash is missing

**Pull request merged**